test/kubemark/resources/hollow-node_template.yaml

```diff
@@ -117,7 +117,7 @@ spec:
           mountPath: /var/log
         resources:
           requests:
-            cpu: 20m
+            cpu: {{npd_cpu_milli}}m
```

Comment:
If we're introducing vars for these, let's also introduce one for memory...

Reply:
Done.